### DISCUSSION TITLE
How can I get Obtainium to link to the TeamViewer APK?

### DISCUSSION THREAD

**Discussion by LearningAsIGo71** [ImranR98/Obtainium#596] [Q&A] [answered]
TeamViewer has direct download links to their APK files on their website, but when I put the URL into Obtainium (e.g. https://download.teamviewer.com/download/TeamViewerQS.apk ), Obtainium responds, "Could not find a suitable release." Is there a way to get this to work?

**Comment by DwainZwerg** [comment]
Probably Obtainium is not able to scrape the apks from the .html page. It is best to report this as an issue.

**Comment by ImranR98** [comment, marked as answer]
You need to add the link to the download page, not the APK itself. So this will work: https://www.teamviewer.com/en-ca/download/android/

There are multiple APKs on the page so Obtainium will automatically pick the top one. In this case, that's `TeamViewerQS.apk` which is probably not the one you want, so you can use the APK filter option to specify the one you want.

Keep in mind that you'll never get notified of app updates, because for HTML sources, Obtainium assumes that the file name will change per release (file name is treated as the "version"). This doesn't seem to be the case with teamviewer.

**Reply by DwainZwerg** [reply]
When is this feature actually coming for APKPure?

**Reply by ImranR98** [reply]
Not sure what you mean - apkpure is already supported (but it looks like they no longer have teamviewer apps)

**Reply by DwainZwerg** [reply]
I mean the toggle (cf. https://github.com/ImranR98/Obtainium/issues/598)

**Reply by ImranR98** [reply]
No specific date planned, I'll look at it at some point
